### ISSUE TITLE
[💡 Feature Request]: Discard duplicate queries when multiple devices are in the same room

### ISSUE THREAD

**Issue by DatDraggy** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#613]
### What's the Problem?

First of all I want to say this project is amazing. I'd had been wanting to run google assistant on windows for a while but the last time I checked it wasn't possible. Thank you so much for your effort!

When there's more than one google assistant (for example my phone and a google nest mini) that heard my Hey Google query, all but one will discard and stop listening. 
Is it possible to add this feature to the desktop client too? Now when I use the hey google wake word, my phone or home mini will also run the query, resulting in it being run two times.

### What's your solution or idea?

Add two options in the settings to either make the desktop client stop processing when another device heard me or make others stop listening and process the request on it's own, if possible.

### Alternatives Considered

Disabling the wake word on desktop

### App Version

Built from source

### OS Version

Windows 10 Pro 21H2

### Additional context

There is no option for the v1 release in the App Version. I installed it from winget today.

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%E2%9C%A8+enhancement%22+) and didn't find any duplicate issue
- [X] This issue only covers one particular feature request

**Comment by TheCre8r** [comment]
In the interim it is possible to disable Hey Google / OK Google in the settings, but I agree this would be helpful.

![image](https://user-images.githubusercontent.com/6027150/148713024-b2f7570b-f45e-4b76-bf6d-8837b2ee687f.png)

**Comment by Melvin-Abraham** [comment]
@DatDraggy I don't think it will be possible to do it right away as the Google Assistant SDK probably doesn't provide such a granular information at the moment. Currently, it's not integrated with Google Home itself for the same reason...

**Comment by DatDraggy** [comment]
@Melvin-Abraham I see. I guess I'll stick to hotkey and only virtual audio input without my mic. Thank you for the reply ^^
